src/proto/beats.proto

```diff
@@ -10,22 +11,75 @@ enum Source {
   USER = 2;
 }
 
-message Beat {
-  optional int32 frame_position = 1;
-  optional bool enabled = 2 [ default = true ];
-  optional Source source = 3 [ default = ANALYZER ];
+enum SectionType {
+  INTRO = 0;
+  OUTRO = 1;
+  OTHER = 2;
```

Comment:
OTHER should be the first value, followed by the specified variants. The first enum value is also the default, not need to specify it redundantly (see below).

Reply:
Do we really need other? And the even intro and outro.
First section is intro, last section is outro, all others are other?

Reply:
We discussed that earlier as well. The first section might not be the intro. We may resume the discussion on Zulip if there are doubts on this.